### PR TITLE
Updating shadow for mobile on involve page

### LINKED ISSUE #183: Title Shadow Mobile Responsive
image

- The text shadows are a bit harder to read on mobile.
- This is because of the shadow's offset being too big on mobile.
- Refer to this doc: https://www.hyperui.dev/blog/text-shadow-with-tailwindcss

### PR DESCRIPTION
**Pull request by ALE00111**
image

### CODE REVIEW

File: src/components/Title.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ const Title = ({ text, food, color }) => {
     <div className="flex flex-row justify-center">
       <div className="inset-0 flex justify-center items-top p-3">
         <div
-          className={`text-shadow ${darkColor} font-omc text-5xl md:text-8xl mb-2 drop-shadow left-1 bottom-0.5 ${lightColor} text-center`}
+          className={`md:text-shadow [text-shadow:_-3px_4px_0px_var(--tw-shadow-color)]  ${darkColor} font-omc text-5xl md:text-8xl mb-2 drop-shadow left-1 bottom-0.5 ${lightColor} text-center`}
```
Comment (seanquiambao) [inline src/components/Title.jsx]:
```suggestion
          className={`md:text-shadow text-shadow-sm ${darkColor} font-omc text-5xl md:text-8xl mb-2 drop-shadow left-1 bottom-0.5 ${lightColor} text-center`}
```

**Review by seanquiambao — CHANGES_REQUESTED** [review]
also check the tailwind.config file.

textShadow "sm" is formatted as 0 1px 2px ...(etc etc..)
the first two numbers are the offset shadows.
the second one is the "blurriness" of the shadow (should be set to 0).

**Review by seanquiambao — CHANGES_REQUESTED** [review]
image


title shadow should not be blurry. check comment above

**Comment by ALE00111** [comment]
Oh sorry I didn't understand the first comment fully. I think that's how it should be after changing the config file.

File: tailwind.config.js [hunk 2]
```diff
@@ -50,7 +50,7 @@ module.exports = {
         blobs1: "url('/assets/blobs1.svg')",
       },
       textShadow: {
-        sm: "0 1px 2px var(--tw-shadow-color)",
+        sm: "0 3px 0px var(--tw-shadow-color)",
```
Comment (seanquiambao) [inline tailwind.config.js]:
```suggestion
        sm: "-3px 3px 0px var(--tw-shadow-color)",
```

**Review by seanquiambao — CHANGES_REQUESTED** [review]

**Review by seanquiambao — APPROVED** [review]

**Pull request merged**